io.edgehog.devicemanager.AgentInfo.json

```diff
@@ -0,0 +1,24 @@
+{
+  "interface_name": "io.edgehog.devicemanager.AgentInfo",
+  "version_major": 0,
+  "version_minor": 1,
+  "type": "properties",
+  "ownership": "device",
+  "mappings": [
+    {
+      "endpoint": "/name",
+      "type": "string",
+      "description": "Name of the agent. Example value: `edgehog-esp32-device`"
+    },
+    {
+      "endpoint": "/url",
+      "type": "string",
+      "description": "URL to the agent code. Example value: `https://github.com/edgehog-device-manager/edgehog-esp32-device`"
+    },
+    {
+      "endpoint": "/version",
+      "type": "string",
+      "description": "Version of the agent. Example value: `0.5`"
+    }
```

Comment:
We might add a further endpoint for runtime (e.g. `/runtime`), that might be "esp-idf VERSION", "Rust 1.58" or maybe "Java 8". What do you think about this?
CC @rbino

Reply:
Given I'd call the whole thing `Runtime`, the information you're talking about could go under `/environment` or `/platform` imho

Reply:
It could be useful, though a little redundant if we plan to use the `URL` field to link to GitHub.

Reply:
> Given I'd call the whole thing `Runtime`, the information you're talking about could go under `/environment` or `/platform` imho

I personally prefer `/environment` if it's ok for both of you